base/sparse/cholmod.jl

```diff
@@ -576,7 +617,7 @@ for Ti in IndexTypes
             d
         end
 
-        function spsolve{Tv<:VTypes}(sys::Integer, F::Factor{Tv,$Ti}, B::Sparse{Tv,$Ti})
+        function solve{Tv<:VTypes}(sys::Integer, F::Factor{Tv,$Ti}, B::Sparse{Tv,$Ti})
```

Comment:
I agree there's merit in that, but it would have meant duplicating https://github.com/JuliaLang/julia/blob/137b19ef2efbba89bf17dd47b57f4d1d996a0c58/base/sparse/cholmod.jl#L1192-L1224. However, I can do that if you prefer.